src/TestableIO.System.IO.Abstractions/IFileSystemAclSupport.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Runtime.Versioning;
+
+namespace System.IO.Abstractions
+{
+    /// <summary>
+    ///     Support ACL functionality on file system instances.
+    /// </summary>
+    public interface IFileSystemAclSupport
+    {
+        /// <summary>
+        ///     Gets a access control object that encapsulates the access control list (ACL) entries for the file or directory in the file system.
+        /// </summary>
+        [SupportedOSPlatform("windows")]
+        object GetAccessControl();
+
+        /// <summary>
+        ///     Gets a access control object that encapsulates the access control list (ACL) entries for the file or directory in the file system.
+        /// </summary>
+        /// <param name="includeSections">One of the <see cref="AccessControlSections" /> values that specifies the type of access control list (ACL) information to receive.</param>
+        [SupportedOSPlatform("windows")]
```

Comment:
I think we can get rid of this attribute as we have it on the class?

```suggestion
```

Reply:
I removed all these attributes in the interface...